service/web/stream/request.go

```diff
@@ -57,14 +62,18 @@ func NewParsedStreamingPullRequest(req *metrov1.StreamingPullRequest) (*ParsedSt
 // NewParsedAcknowledgeRequest ...
 func NewParsedAcknowledgeRequest(req *metrov1.AcknowledgeRequest) (*ParsedStreamingPullRequest, error) {
 	parsedReq := &ParsedStreamingPullRequest{}
+	ctx := context.Background()
```

Comment:
nope. Use the calling functions context.

Reply:
Remove this as no longer required after removing log statements from model file